app/api/data_types.ts

```diff
@@ -0,0 +1,51 @@
+export enum DRIVER_STATION {
```

Comment:
Instead of using numeric enums, since this data will be serialized, I recommend setting these to strings

Reply:
fixed !!